packages/@react-aria/interactions/src/usePress.ts

```diff
@@ -740,5 +740,17 @@ function shouldPreventDefault(target: Element) {
 
 function isVirtualPointerEvent(event: PointerEvent) {
   // If the pointer size is zero, then we assume it's from a screen reader.
-  return event.width === 0 && event.height === 0;
+  // Android TalkBack double tap will sometimes return a event with width and height of 1
+  // and pointerType === 'mouse' so we need to check for a specific combination of event attributes.
+  return (
+    (event.width === 0 && event.height === 0) ||
+    (event.buttons === 1 &&
+      event.width === 1 &&
+      event.height === 1 &&
+      event.type === 'pointerdown' &&
+      event.pressure === 0 &&
+      event.pointerType === 'mouse' &&
+      event.detail === 0
```

Comment:
Made this super specific to the Android TalkBack double tap event so that we hopefully don't run into an issue where other pointer events are mistaken as virtual pointer events. A brief rundown:
- Desktop click events share the same `width`, `height`, `type: pointerdown`, and `pointerType === mouse` as the Android TalkBack double tap. The above attributes (i.e. pointerType and pressure) however help distinguish the TalkBack event from touch events
- For devices that don't support pressure measurements, their pointer event usually return .5 matching the following spec: https://w3c.github.io/pointerevents/#dom-pointerevent-pressure. Unfortunately this doesn't happen for Safari pointer events which return `pressure: 0` on touch and desktop
- Mouse clicks in Safari are only distinguishable from the Android TalkBack double tap events via `detail` (i.e. `detail: 1` vs `detail: 0`

Technically I could minimize the check above to be `event.detail === 0 && event.pointerType === 'mouse'` but making it more specific felt safer

Reply:
Thanks @reidbarber for helping me check, stylus/pen returns `pointerType === 'pen' this check should be ok as well

Reply:
The part that seems the most concerning to me to rely on is `pointerType === 'mouse'`. It seems like a bug that on a touchscreen device it would emit a mouse event, and I fear that this will be fixed and we will be broken again. Is `width === 1 && height === 1 && detail === 0 && pressure === 0` enough? I think the width/height should eliminate real touch events as well as all screen readers other than TalkBack, detail === 0 should eliminate Safari mouse events, and pressure === 0 should eliminate chrome. My Apple Pencil reports events with 0.5 width and height, and a non-zero pressure as well. Any other cases I missed?

Reply:
Yep, I think that should be all since FF/Edge will have pressure === .5 as well. I'm fine w/ `width === 1 && height === 1 && detail === 0 && pressure === 0`, was just wary of a touch event that coincidentally had `width/height ===1` but with the level of precision that those attributes get measured its probably ok haha.